docs/config/README.md

```diff
@@ -1380,7 +1380,7 @@ AA -------------------------------------------- BB -----------------------------
 ## Google Cloud (`gcloud`)
 
 The `gcloud` module shows the current configuration for [`gcloud`](https://cloud.google.com/sdk/gcloud) CLI.
-This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gcloud/configurations/config_{CONFIG NAME}` file and the `CLOUDSDK_CONFIG` env var.
+This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gcloud/configurations/config_{CONFIG NAME}` file and the `CLOUDSDK_CONFIG` env var. Setting `active` to `"NONE"` will disable the module.
```

Comment:
Trying to read up on the "NONE" config mentioned. I'm not sure if it's `active = "NONE"` or if it's the active *name* is `"NONE".

So maybe:

```suggestion
This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gcloud/configurations/config_{CONFIG NAME}` file and the `CLOUDSDK_CONFIG` env var. Setting active name to `"NONE"` will disable the module.
```

Reply:
```suggestion
This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gcloud/configurations/config_{CONFIG NAME}` file and the `CLOUDSDK_CONFIG` env var. If the active config name in `~/.config/gcloud/active_config` is set to `"NONE"`, the module will be disabled.
```